### ISSUE TITLE
Won't start on windows 8.1 x64

### ISSUE THREAD

**Issue by Raphi111** [spyder-ide/spyder#2934]
Here is what I did:

Install python 3.4.4 (amd64)
Install lastest PyQt4 (x64)
Install spyder 2.8.4 (for python 3.4)

When I open a command line and launch spyder.bat, I get an image that opens and close immediately, and the command line closes itself.
I can't provide more information by myself as the terminal closes itself.
python3.4 is already in my "path" variable.

**Comment by ccordoba12** [comment]
You don't need to launch `spyder.bat` directly. You just need to open a terminal and run

    spyder --show-console

to avoid the console being closed and see what errors appear there.

However, if you're new to Python, we'd recommend you to follow our installation instructions:

http://pythonhosted.org/spyder/installation.html

and use a Scientific Python Distribution (like Anaconda or WinPython) to install Spyder, instead of doing what you're doing right now :-)

**Comment by Raphi111** [comment]
Thanks for your help, I tried it already but running "spyder" in a terminal returns "command not found" error.
Although I'm new to Python, I don't want to install a whole distribution as I don't need any of the extra packages they are offering.

**Comment by ccordoba12** [comment]
Well, if you can't run `spyder` you're already in troubles. That means that the `Scripts` directory is not in your PATH, which points to a problem with your installation.

I'd sincerely recommend you to uninstall what you have installed so far, and better install Miniconda (the lightweight version of Anaconda):

http://conda.pydata.org/miniconda.html

and then run this command in a terminal

    conda install spyder

That will not only install Spyder but all its dependencies (something you're not doing with your current procedure).

If you persist in your current path, I'm afraid you're in your own to debug these problems :-)

**Comment by Raphi111** [comment]
After adding the "script" directory to my PATH variable, I'm able to launch spyder directly from the terminal with the --show-console parameter and here is what I get:

```
Traceback (most recent call last):
  File "C:\Program Files\Python34\Scripts\spyder", line 3, in <module>
    start_app.main()
  File "c:\program files\python34\lib\site-packages\spyderlib\start_app.py", lin
e 115, in main
    spyder.main()
  File "c:\program files\python34\lib\site-packages\spyderlib\spyder.py", line 2
278, in main
    app = initialize()
  File "c:\program files\python34\lib\site-packages\spyderlib\spyder.py", line 2
134, in initialize
    rope_patch.apply()
  File "c:\program files\python34\lib\site-packages\spyderlib\rope_patch.py", li
ne 131, in apply
    from rope.contrib import codeassist
  File "c:\program files\python34\lib\site-packages\spyderlib\utils\external\rop
e\contrib\codeassist.py", line 9, in <module>
    from rope.contrib import fixsyntax
  File "c:\program files\python34\lib\site-packages\spyderlib\utils\external\rop
e\contrib\fixsyntax.py", line 29
    except exceptions.ModuleSyntaxError, e:
                                       ^
SyntaxError: invalid syntax
```
What I don't understand is that I did everything written on the installation page (https://pythonhosted.org/spyder/installation.html#the-hard-way) and installed all the essential requirements but it still won't run.

**Comment by ccordoba12** [comment]
Yes, but the hard way is for experts, not newbies (hence its title ;-).

In any case, you hit a bug in our installers for Python 3 (we already fixed this for our next major version, Spyder 3.0, in case you want to complain more :-).

To fix it, you need to remove this folder:

    C:\program files\python34\lib\site-packages\spyderlib\utils\external

And please don't forget to install the rest of our dependencies using `pip`. The installer doesn't come with them, and so if you don't install them you won't get any of the benefits Spyder has to offer. This is another thing we plan to fix in 3.0.

**Comment by ccordoba12** [comment]
Closing as a duplicate of issue #2648.
